Dragonflight/ShamanRestoration.lua

```diff
@@ -206,6 +206,25 @@ spec:RegisterAuras( {
         duration = 1800,
         max_stack = 1
     },
+	high_tide = {
+        id = 288675,
+        duration = 25,
+        max_stack = 2
+    },
+	cloudburst_totem = {
+        id = 157504,
+        duration = 18,
+        max_stack = 1
+    },
+	natures_swiftness = {
```

Comment:
This exists in ShamanElemental.lua and will be inherited automatically.  Was there a reason to add it?

Reply:
Yeah, you are correct. IDK why it wasn't working on the first tests so I added natures_swiftness there. Removed these lines.